### PR TITLE
changed Correct By behaviour + fixed receipt bug + fixed affix

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#2830

*Description of changes:*
- don't assign initial Corrected By
- added receipt code for correction filings
- fixed affix scrolling behaviour

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/FilingHistoryList.vue [hunk 1]
```diff
@@ -324,7 +324,6 @@ export default {
               name: 'Annual Report',
               documentName: `${this.entityIncNo} - ${title} - ${filingDate}.pdf`
             }],
-            paperOnly: false,
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList.vue]: This is not needed and assigning it is inconsistent with other code.

File: src/components/Dashboard/FilingHistoryList.vue [hunk 2]
```diff
@@ -383,14 +381,19 @@ export default {
...
+
         const item = {
           type: filing.header.name,
+          title: `Correction - ${this.typeToTitle(filing.correction.correctedFilingType)}`,
+          filingId: filing.header.filingId,
           filingAuthor: filing.header.certifiedBy,
+          filingDateTime: localDateTime,
           filingDate: filing.correction.correctedFilingDate,
-          filingId: filing.header.filingId,
+          isCorrection: true,
```
Comment (cameron-freshworks) [inline src/components/Dashboard/FilingHistoryList.vue]: This ones new, where do we use this property?

Comment (cameron-freshworks) [reply src/components/Dashboard/FilingHistoryList.vue]: Appears it is for future use in `correctThisItem()` ?

Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList.vue]: Yes, for future use.

**Pull request merged**